dev-docs/development-process.md

```diff
@@ -195,6 +195,15 @@ HAIL_DEFAULT_NAMESPACE=default hailctl dev deploy -b <github_user_name>/hail:<br
 After this dev deploy completes, you should be able to access your namespace
 by navigating to https://internal.hail.is/<username>/batch.
 
+The service accounts used in developer namespaces do not have permission to create pet service accounts in the
+`hail-vdc` project, so the gsa-key secrets for these must be copied across from the default namespace in order
+to run real jobs on the developer namespaces' batch services:
+```bash
+$ download-secret <username>-gsa-key
+$ mv secret.json key.json
+$ kubectl create secret generic <username>-gsa-key --namespace=<username> --from-file=key.json
+```
```

Comment:
This is probably my k8s naivete showing, but what is the `mv` for? Does the source path in `kubectl create secret` matter?

Reply:
So I didn't actually realize this (I copied these commands from @ehigham ) but yes - apparently the filename becomes the default key name for the secret. So, thanks for asking, because TIL!

Reply:
I believe you can set the secret name too, or download the secret to another file name. Look at `download-secret` and `create-secret` in `devbin/functions.sh` (or similar suffix)